lib/libyear_bundler/report.rb

```diff
@@ -113,8 +109,38 @@ def put_summary(summary)
       elsif @options.releases?
         put_sum_seq_delta_summary(summary[:sum_seq_delta])
       elsif @options.libyears?
-        put_libyear_summary(summary[:sum_years])
+        put_libyear_summary(summary[:sum_libyears])
       end
     end
+
+    def sum_libyears(gem, memo)
+      memo[:sum_libyears] += gem.libyears
+    end
```

Comment:
What do you think about `increment_` instead of `sum_`, for all of these methods?

Reply:
I dig it.

Reply:
On second thought, we should probably extract these kinds of methods out of `Report`, which is longer concerned just with presentation as the top-level class comment implores.

Reply:
"implores" 😁

Reply:
:) Timestamp: 12:44 AM